src/Plugin/WebformElement/NetsEasyPaymentElement.php

```diff
@@ -126,4 +134,29 @@ public function alterForm(array &$element, array &$form, FormStateInterface $for
     }
   }
 
+  /**
+   * Get recipient elements.
+   *
+   * @phpstan-return array<string, mixed>
+   */
+  private function getRecipientElements(): array {
```

Comment:
Hide your tracks when stealing code!

```suggestion
  private function getAmountElements(): array {
```